### DISCUSSION TITLE
React version?

### DISCUSSION THREAD

**Discussion by spacenerd7** [nasa/openmct#5315] [Community Q&A]
Hi! Are there any plans to offer a React version of this code base?  I thought I read that several months ago, but am unable to re-locate that information.  Would love to collaborate on something like this if possible.

**Comment by MarkCodering** [comment]
React is a great tool for building a front-end interface. Since React is a library instead of a web development framework like Vue.js or Angular, how would approach this problem or do you have an explicit idea on how to use React.js with NASA Open MCT?
